codalab/client/local_bundle_client.py

```diff
@@ -530,21 +511,75 @@ def select_unreadable_worksheets(uuids):
 
         return bundle_dict
 
-    # Return information about an individual target inside the bundle.
+    def check_download_permission(self, target):
```

Comment:
The reason for the separate method is because I need to check permission in bundle_rpc_server and I don't want to access client.model and client._current_user() from there.